### PR TITLE
Scaffold mobile app

### PR DESCRIPTION
**Pull request by raomanasa**
[PT-LINK](https://www.pivotaltracker.com/story/show/170925482)

USER STORY
As a visitor
In order to get my news on the go
I want to be able to use a mobile app for the Herald

- Setup Repo on Craft Academy github for mobile app.
- Scaffold the app
- Remove unnecessary scaffolded code.

Screenshot 2020-01-29 at 9 34 32 AM

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Redirect, Route } from 'react-router-dom';
+import { IonApp, IonRouterOutlet } from '@ionic/react';
+import { IonReactRouter } from '@ionic/react-router';
+import Home from './pages/Home';
+import '@ionic/react/css/core.css';
+import '@ionic/react/css/normalize.css';
```
Comment (mantanquzzmo) [inline src/App.jsx:7]: are these necessary?

File: src/pages/Home.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonTitle>Ionic Blank</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent className="ion-padding">
+        The world is your oyster.
+        <p>
```
Comment (mantanquzzmo) [inline src/pages/Home.jsx:13]: can clean up this as well. Leave maybe our title

File: src/App.jsx (hunk 1 above)
Comment (raomanasa) [reply src/App.jsx:7]: - Normalize.css makes browsers render all elements more consistently and in line with modern standards. It precisely targets only the styles that need normalizing.
-  core .css import of the core CSS for Ionic .
- Depending on the furthur necessaries we can import or deleate the necessary components.

File: src/pages/Home.jsx (hunk 2 above)
Comment (raomanasa) [reply src/pages/Home.jsx:13]: ok

**Pull request merged**